packages/oui-datagrid/src/index.spec.js

```diff
@@ -636,6 +636,40 @@ describe("ouiDatagrid", () => {
             const secondDatagridRows = getRows(angular.element(datagridElements[1]));
             expect(secondDatagridRows.length).toEqual(secondDatagridSize);
         });
+
+        describe("Columns", () => {
+            it("should support dataset notation", () => {
+                const element = TestUtils.compileTemplate(`
+                        <oui-datagrid data-rows="$ctrl.rows">
+                            <oui-column data-property="firstName"
+                                data-sortable="asc"
+                                data-title="'First name'"></oui-column>
+                            <oui-column property="lastName"
+                                data-sortable
+                                data-title="'Last name'"></oui-column>
+                            <oui-action-menu>
+                                <oui-action-menu-item text="Action 1"></oui-action-menu-item>
+                                <oui-action-menu-item text="Action 2"></oui-action-menu-item>
+                            </oui-action-menu>
+                        </oui-datagrid>
+                    `, {
+                        rows: fakeData.slice(0, 5)
+                    }
+                );
+
+                // Check data-sortable and data-title
+                const headerCells = getHeaderRow(element).find("th");
```

Comment:
You can do this instead, to correspond to what I already do before (unless I missed something):

```javascript
const $headerRow = getHeaderRow(element);

expect(isSortableCell(getHeaderCell($headerRow, 0))).toBe(true);
expect(getHeaderCell($headerRow, 0).html()).toEqual("First name");
expect(getHeaderCell($headerRow, 0).class()).toContain("oui-datagrid__header_sortable-asc");
```

Reply:
done